lib/VMwareWebService/MiqPbmInventory.rb

```
@@ -8,6 +8,7 @@ def pbm_initialize(vim)
       @pbm_svc = PbmService.new(vim) if apiVersion >= '5.5' && isVirtualCenter
     rescue => err
       $vim_log.warn("MiqPbmInventory: Failed to connect to SPBM endpoint: #{err}")
+      raise err.to_s
```

If you want to just re-raise the exception you can just call `raise` you don't need to give it any args